concent_api/core/tests/utils.py

```diff
@@ -686,12 +708,12 @@ def _get_deserialized_task_failure(
 
     def _get_deserialized_reject_report_computed_task(
         self,
-        timestamp           = None,
-        cannot_compute_task = None,
-        task_failure        = None,
-        task_to_compute     = None,
-        reason              = None,
-    ):
+        timestamp: Union[str, datetime.datetime, None] = None,
```

Comment:
only this is optiional, the rest cannot be `None`

Reply:
done

Reply:
as agreed, `task_to_compute` and `reason` cannot be None, the rest can be